docs/skopeo.1.md

```diff
@@ -29,10 +33,20 @@ Most commands refer to container images, using a _transport_`:`_details_ format.
   By default, uses the authorization state in `$HOME/.docker/config.json`,
   which is set e.g. using `(docker login)`.
 
+  **docker-archive:**_docker-reference_
+  An image _tag_ stored in the local docker archives
```

Comment:
**docker-archive:**_path_[**:**_docker-reference_]

An image stored in a `docker save`-formatted file. _docker-reference_ is used only when creating such file, and it must not contain a digest.